pkgs/development/libraries/updfparser/default.nix

```diff
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation rec {
   name = "updfparser";
-  version = "unstable-2023-01-10";
-  rev = "a421098092ba600fb1686a7df8fc58cd67429f59";
+  version = "unstable-2023-08-10";
```

Comment:
Since the commit is from `2023-08-08` we should name the version accordingly

Reply:
Good point. Fixed version.